src/_main_/utils/utils.py

```diff
@@ -3,72 +3,92 @@
+    defined in there and returns them as a list.
+    """
+    return [
+        m[1]
+        for m in inspect.getmembers(models, inspect.isclass)
+        if (isinstance(m[1], Base.ModelBase))
+    ]
 
 
 def get_models_and_field_types(models):
-  """
-  This method take a models.py class and makes a dictionary of all the models
-  mapping them to their fields in groups.  
-
-  eg. {
-        model1: {"m2m": {...}, "fk": {....}, "other":{....}, 
-          "required_fields":{.....}}
-        ....
-      }
-      Hence for each model, we collect and group all the many to many fields
-      as well as foreignkeys as well as get which fields are required
-  """
-  all_models = get_all_models(models)
-  result = {}
-  for m in all_models:
-    result[m]={
-      "m2m": set(),"fk":set(), "other": set(), 
-      "required_fields": set(), "all_fields": set()
-    }
-    for f in m._meta.get_fields():
-      result[m]["all_fields"].add(f.name)
-      if isinstance(f, ManyToManyField):
-        result[m]["m2m"].add(f.name)
-      elif isinstance(f, ForeignKey):
-        result[m]["fk"].add(f.name)
-      else:
-        result[m]["other"].add(f.name)
-
-      if hasattr(f, 'blank') and f.blank == False:
-        result[m]["required_fields"].add(f.name)   
-  return result
-
-def strip_website(url: str)-> str:
-  if not url:
-    return None 
-  url = url.replace("http://","")
-  url = url.replace("https://","")
-  url = url.split("/")[0] # deal with trailing spaces
-  return url.strip()
\ No newline at end of file
+    """
+    This method take a models.py class and makes a dictionary of all the models
+    mapping them to their fields in groups.
+
+    eg. {
+          model1: {"m2m": {...}, "fk": {....}, "other":{....},
+            "required_fields":{.....}}
+          ....
+        }
+        Hence for each model, we collect and group all the many to many fields
+        as well as foreignkeys as well as get which fields are required
+    """
+    all_models = get_all_models(models)
+    result = {}
+    for m in all_models:
+        result[m] = {
+            "m2m": set(),
+            "fk": set(),
+            "other": set(),
+            "required_fields": set(),
+            "all_fields": set(),
+        }
+        for f in m._meta.get_fields():
+            result[m]["all_fields"].add(f.name)
+            if isinstance(f, ManyToManyField):
+                result[m]["m2m"].add(f.name)
+            elif isinstance(f, ForeignKey):
+                result[m]["fk"].add(f.name)
+            else:
+                result[m]["other"].add(f.name)
+
+            if hasattr(f, "blank") and f.blank == False:
+                result[m]["required_fields"].add(f.name)
+    return result
+
+
+def strip_website(url: str) -> str:
+    if not url:
+        return None
+    url = url.replace("http://", "")
+    url = url.replace("https://", "")
+    url = url.split("/")[0]  # deal with trailing spaces
+    return url.strip()
+
+
+class Console:
```

Comment:
Not sure what this is about?  Just for local testing?

Reply:
Yes, it wont affect the product...  🤣 😂😂😂😂 just makes it easy to find the things I print.... the terminal logs too many things.

 Also I think I will just close this PR in the mean time... I am still working on it and I forgot you would be notified of the tiny changes 🤣 🤣 ....